### PR TITLE
ollama: Fix JSON format bug issue when not streaming

### PR DESCRIPTION
**Pull request by doslindos**
Fix issue #889 

### PR Checklist

- [x] Read the [Contributing documentation](https://github.com/tmc/langchaingo/blob/main/CONTRIBUTING.md).
- [x] Read the [Code of conduct documentation](https://github.com/tmc/langchaingo/blob/main/CODE_OF_CONDUCT.md).
- [x] Name your Pull Request title clearly, concisely, and prefixed with the name of the primarily affected package you changed according to [Good commit messages](https://go.dev/doc/contribute#commit_messages) (such as `memory: add interfaces for X, Y` or `util: add whizzbang helpers`).
- [x] Check that there isn't already a PR that solves the problem the same way to avoid creating a duplicate.
- [x] Provide a description in this PR that addresses **what** the PR is solving, or reference the issue that it solves (e.g. `Fixes #123`).
- [x] Describes the source of new concepts.
- [x] References existing implementations as appropriate.
- [x] Contains test coverage for new functions.
- [x] Passes all [`golangci-lint`](https://golangci-lint.run/) checks.

### CODE REVIEW

**Comment by tmc** [comment]
Thanks! I simplified things a bit and made the example a bit more robust.

File: llms/ollama/internal/ollamaclient/types.go [hunk 1]
```diff
@@ -49,7 +49,7 @@ type Message struct {
 type ChatRequest struct {
 	Model     string     `json:"model"`
 	Messages  []*Message `json:"messages"`
-	Stream    *bool      `json:"stream,omitempty"`
+	Stream    bool       `json:"stream,omitempty"`
```
Comment (corani) [inline llms/ollama/internal/ollamaclient/types.go:52]: This makes the `omitempty` meaningless, because bools can't be "empty".

Comment (tmc) [reply llms/ollama/internal/ollamaclient/types.go:52]: that's not quite right, in Go json encoding: `The "omitempty" option specifies that the field should be omitted from the encoding if the field has an empty value, defined as false, 0, a nil pointer, a nil interface value, and any empty array, slice, map, or string.`

Comment (corani) [reply llms/ollama/internal/ollamaclient/types.go:52]: hmm, then perhaps I'm mistaken... I recall seeing `"field": false` in JSON output despite `omitempty`, but I haven't tested it so I may be misremembering :)

**Pull request merged**